fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -262,17 +262,11 @@ ACTOR Future<Void> getRate(UID myID, Reference<AsyncVar<ServerDBInfo>> db, int64
 			nextRequestTimer = Never();
 			bool detailed = now() - lastDetailedReply > SERVER_KNOBS->DETAILED_METRIC_UPDATE_RATE;
 
-			TransactionTagMap<uint64_t> tagCounts;
-			for(auto itr : *throttledTags) {
-				for(auto priorityThrottles : itr.second) {
-					tagCounts[priorityThrottles.first] = (*transactionTagCounter)[priorityThrottles.first];
-				}
-			}
```

Comment:
It sees something has gone wrong here, possibly with a merge. These tag counts were supposed to be part of the `GetRateInfoRequest`.

Reply:
As I see, the ratekeeper also doesn't use tagCount? I remember `ratekeeper.actor.cpp` doesn't have non-trivial changes before this PR.

Reply:
`tagCounts` replaced the `*transactionTagCounter` argument in this PR: https://github.com/apple/foundationdb/pull/3437